src/RequestHandler.ts

```diff
@@ -105,6 +105,7 @@ async function handleFetchAsPromise<T extends HttpMethodTemplate>(
     // This error only occurs when there are network errors and such
     throw opts.errorWithCleanerStack.addContext(context).setCause(e);
   }
+  const contextIncludingBody = { ...context, body: response.statusText };
```

Comment:
So, above, we have `context`, then `errorContext`, and now we're adding `contextIncludingBody`. In the function below, we have a similar structure and but we just have `errorContext`.
I think we can get away with renaming `context` => `errorContext`, and then just adding properties like, body and statusCode to that.

Reply:
Also, I had a quick chat with Ivan. He pointed out that we can just add the `response.body`  here and not the `response.statusText`.
We looked at the Response object, it extends the Body object, which has the body. So we also have access to it here.

Reply:
Fair enough, we can just use the `statusText` for the body. I would then give the field a diffenent name (eg. message) since it isn't actually the body that we send back.
Another thing is to get to body from the response I found out that we need to parse the 'stream'. So what you have above, `response.body`, is a stream. To read it we can use `await response.json()`. That actually gives us back the body.